config/initializers/paperclip.rb

```diff
@@ -0,0 +1,2 @@
+Paperclip::Attachment.default_options[:url] = ':s3_domain_url'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.